tests/tests_pytorch/strategies/test_ddp_integration.py

```diff
@@ -202,22 +201,20 @@ def training_step(self, batch, batch_idx):
         return super().training_step(batch, batch_idx)
 
 
-@pytest.mark.skipif(
-    # TODO: investigate threading issue in this configuration
-    _IS_WINDOWS,
-    # and (sys.version_info.major, sys.version_info.minor) == (3, 11)
-    # and compare_version("torch", operator.eq, "2.1.0", use_base_version=True)
-    reason="threading issue",
-)
-def test_find_unused_parameters_exception():
+@RunIf(standalone=True)
```

Comment:
Couldn't find the "why" to this mystery, but standalone seems to work and doesn't require us to skip.